lib/virt_autotest/utils.pm

```diff
@@ -140,6 +140,37 @@ sub install_default_packages {
     zypper_call '-t in nmap iputils bind-utils', exitcode => [0, 4, 102, 103, 106];
 }
 
+sub ensure_online {
```

Comment:
I seem to see a similar function ever created by @waynechen55 , which starts guests when they are not running. Wayne, can you help to share it here? Maybe this ensure_online function can be done by extending that, so as to avoid similar functions here and there?

Reply:
I'm also interested of such function. I think we really need to have one, but now I'm not aware of any.

Reply:
Hey! As I will be away for two weeks from next week onwards, I would like to push this forward. If no stopper arises, it would be great to merge this PR tomorrow :slightly_smiling_face:

Reply:
> Maybe this ensure_online function can be done by extending that, so as to avoid similar functions here and there?

About guest start or restart, there is one called perform_guest_restart in tests/virt_autotest/virt_utils.pm.

Reply:
@alice-suse this is not about starting or restarting a guest but ensuring it is online.
Anyways thank you for pointing it out, I'll reuse `perform_guest_restart()` in my next PR.